src/main/java/com/github/princesslana/eriscasper/util/SendableImage.java

```diff
@@ -0,0 +1,25 @@
+package com.github.princesslana.eriscasper.util;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+import javax.imageio.ImageIO;
+
+public abstract class SendableImage {
```

Comment:
We make a base class and just extend it in one place? Why not just one class?

Reply:
It's extended in 2 places iirc AvatarImage and EmojiImage